data/npc/scripts/ship.lua

```diff
@@ -0,0 +1,30 @@
+local keywordHandler = KeywordHandler:new()
+local npcHandler = NpcHandler:new(keywordHandler)
+NpcSystem.parseParameters(npcHandler)
+
+function onCreatureAppear(cid)              npcHandler:onCreatureAppear(cid)            end
+function onCreatureDisappear(cid)           npcHandler:onCreatureDisappear(cid)         end
+function onCreatureSay(cid, type, msg)      npcHandler:onCreatureSay(cid, type, msg)    end
+function onThink()                          npcHandler:onThink()                        end
+
+keywordHandler:addKeyword({'captain'}, StdModule.say, {npcHandler = npcHandler, text = 'I am the captain of this sailing-ship.'})
+keywordHandler:addKeyword({'trip'}, StdModule.say, {npcHandler = npcHandler, text = 'Where do you want to go? To Trekolt, Rhyves, Varak or Saund?'})
+keywordHandler:addKeyword({'ice'}, StdModule.say, {npcHandler = npcHandler, text = 'I\'m sorry, but we don\'t serve the routes to the Ice Islands.'})
+
+local node1 = keywordHandler:addKeyword({'trekolt'}, StdModule.say, {npcHandler = npcHandler, onlyFocus = true, text = 'Do you seek a passage to Trekolt for 100 gold?'})
```

Comment:
why are you initializing the value called node if it has no use?

Reply:
care to explain what you mean by that? looks alright to me

Reply:
local node1 = keywordHandler:addKeyword...

Reply:
@nekiro Each node has duplicate child keywords (`yes` and `no`) which does different actions depending on the node. (different position). 

But if keywordhandler returns itself/its node, perhaps these can be appended to the same line. 
```lua
keywordHandler:addKeyword({'rhyves'}, ... ):addChildKeyword({'yes'}, ... ):addChildKeyword({'no'}, ... )
-- beautified?
keywordHandler:addKeyword({'rhyves'}, ... )
    :addChildKeyword({'yes'}, ... )
    :addChildKeyword({'no'}, ... )
```